x-pack/plugins/cloud_security_posture/server/create_transforms/latest_findings_transform.ts

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import type { TransformPutTransformRequest } from '@elastic/elasticsearch/lib/api/types';
+import { CSP_KUBEBEAT_INDEX_PATTERN, LATEST_FINDINGS_INDEX_PATTERN } from '../../common/constants';
+
+export const latestFindingsTransform: TransformPutTransformRequest = {
+  transform_id: 'cloud_security_posture.latest-default-0.0.1',
+  description: 'Defines findings transformation to view only the latest finding per resource',
+  source: {
+    index: CSP_KUBEBEAT_INDEX_PATTERN,
```

Comment:
fyi, when `cloudbeat` would support namespaces, this transform would take results from other spaces into the default's latest index.

what if the index pattern of this transform would be `logs-cis_kubernetes_benchmark.findings-default-*`. it might prevent a state that would be harder to fix if get there.

Reply:
I'm pretty sure the pattern you suggested won't work as the datastream name is `logs-cis_kubernetes_benchmark.findings-default` and the underlying index name is `.ds-logs-cis_kubernetes_benchmark.findings-default-2022.04.18-000001`

<img width="1467" alt="Screen Shot 2022-04-18 at 10 33 24" src="https://user-images.githubusercontent.com/63912106/163773550-33bd2f61-c648-4237-9e9f-6850cf7cd9f2.png">
<img width="1461" alt="Screen Shot 2022-04-18 at 10 35 11" src="https://user-images.githubusercontent.com/63912106/163773619-f6801100-7979-450a-b3d6-c55f9fcf8498.png">

Reply:
correct, it looks like the datastream name would achieve the same result though.
anyway, we can fix that later